doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -102,23 +103,78 @@ See the description of each element in the following tabs:
 
          <?xml version="1.0"?>
          <cluster>
-           <domain>General</domain>
-           <name>MyNewCluster</name>
-           <code>0xFFF1FC01</code>
-           <define>MY_NEW_CLUSTER</define>
-           <description>The MyNewCluster cluster showcases a cluster manufacturer extensions</description>
-           <attribute side="server" code="0x00" define="MY_ATTRIBUTE" type="boolean" writable="true" default="false" optional="false">MyAttribute</attribute>
-           <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
-             <description>Command that takes two uint8 arguments and returns their sum.</description>
-             <arg name="arg1" type="int8u"/>
-             <arg name="arg2" type="int8u"/>
-           </command>
-           <event source="server" code="0x01" name="MyEvent" optional="false">
-             <description>Event that is generated by the server.</description>
-             <arg name="arg1" type="int8u"/>
-           </event>
+            <domain>General</domain>
+            <name>MyNewCluster</name>
+            <code>0xFFF1FC01</code>
+            <define>MY_NEW_CLUSTER</define>
+            <description>The MyNewCluster cluster showcases a cluster manufacturer extensions</description>
+            <attribute side="server" code="0x00" define="MY_ATTRIBUTE" type="boolean" writable="true" default="false" optional="false">MyAttribute</attribute>
+            <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
+               <description>Command that takes two uint8 arguments and returns their sum.</description>
+               <arg name="arg1" type="int8u"/>
+               <arg name="arg2" type="int8u"/>
+            </command>
+            <event source="server" code="0x01" name="MyEvent" optional="false">
+               <description>Event that is generated by the server.</description>
+               <arg name="arg1" type="int8u"/>
+            </event>
          </cluster>
 
+   .. tab:: ``<cluster>``
+
+      ``<clusterExtension>`` defines the extension of an existing cluster and consist of the following attributes:
```

Comment:
hmm, it doesn't consist of attributes only. Maybe somehow rewrite this sentence because we actually use the "attribute" term below?